docs/faq.md

```diff
@@ -38,6 +38,16 @@ There are no plans regarding other providers at the moment.
 
 Services exposed via `type=LoadBalancer` and for the hostnames defined in Ingress objects. It also seems useful to expose Services with `type=NodePort` to point to your cluster's nodes directly, but there's no commitment to doing this yet.
 
+### How do I specify DNS name for my Kubernetes objects?
+
+There are three sources of information for ExternalDNS to decide on DNS name. ExternalDNS will pick one in order as listed below: 
+
+1. For ingress objects ExternalDNS will create a DNS record based on the host specified for the ingress object. For services ExternalDNS will look for the annotation `external-dns.alpha.kubernetes.io/hostname` on the service and use the corresponding value. 
+
+2. If compatibility mode is enabled (e.g. `--compatibility={mate,molecule}` flag), External DNS will parse annotations used by Zalando/Mate, Kops DNS Controller, wearemolecule/route53-kubernetes.
```

Comment:
Kops DNS Controller is not yet supported with the listed possible values for this flag. Maybe mention that or take it out?